package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "selfcare-dashboard-groups-microfrontend",
-  "version": "1.0.0",
+  "version": "1.1.1",
```

Comment:
revert this
```suggestion
  "version": "1.0.0",
```

Reply:
@loremedda you are doing a PR into release-uat!
please, review the asked changes and open a new PR